mqtt-simulator/simulator.py

```diff
@@ -11,26 +11,35 @@ def __init__(self, settings_file):
     def load_settings(self, settings_file):
         with open(settings_file) as json_file:
             config = json.load(json_file)
-            self.broker_url = config['BROKER_URL']
-            self.broker_port = config['BROKER_PORT']
+            self.broker_url = config.get('BROKER_URL', 'localhost')
+            self.broker_port = config.get('BROKER_PORT', 1883)
+            broker_time_interval = config.get('TIME_INTERVAL', 10)
+            broker_protocol = config.get('PROTOCOL_VERSION', 4) # mqtt.MQTTv311
+            broker_clean = config.get('CLEAN_SESSION', True)
+            broker_qos = config.get('QOS', 2)
+            broker_retained = config.get('RETAINED', False)
```

Comment:
Just a small note, it would be nice to have these default values documented in the README later

Reply:
Definitely! That is the next step but wanted your feedback first to work through any changes.